content/articles/setting-up-hotspot-on-kali-linux/index.md

```diff
@@ -0,0 +1,149 @@
+ ```bash
+ lsusb
+ ```
+ The output is as follows:
+ ![Drivers](/engineering-education/setting-up-hotspot-on-kali-linux/drivers.png)
+### Installation of the necessary tools
+In order to create our hotspot, we need to install the required tools. The first and crucial tool is **hostapd** which acts as the hotspot server.
+To install **`hostapd`**, type in the following command:
+
+```bash
+ apt-get install hostapd
+ ```
+The second tool will be **dnsmasq** which will act like the server's **dhcp**.
+ To install **`dnsmasq`**, type in the following:
+ 
+```bash
+ apt-get install dnsmaq
+ ```
+Once we're done with the installations, we can now move to the next step.
+We shall proceed to check for the just installed processes. If already started, stop those in progress, and deter them from starting during start-up of the system.
+
+Below are the commands used for stopping and disabling updates of the services:
+
+```bash
+sudo service hostapd stop
+```
+```bash
+sudo service dnsmasq stop
+```
+```bash
+sudo update-rc.d hostapd disable
+```
+```bash
+sudo update-rc.d dnsmasq disable
+```
+The output is as follows:
+
+![Installations](/engineering-education/setting-up-hotspot-on-kali-linux/installations.png)
+
+ ###  Setting up configuration files
+After installing the services, we shall then need to configure the necessary files i.e **`hotspot.sh`**.
+
+We will begin by typing the  command below in our terminal.
+```bash
+sudo gedit /etc/dnsmasq.conf
+```
+After running the command, write the code below under **`hotspot.sh`** for configuration:
+```
+# Only bind it to one interface
+bind-interfaces
+# Select the interface to use for binding
+interface=wlan0
+# Select a scope of IP addresses to be used in DHCP leasing
+
+dhcp-range=192.168.1.1,192.168.1.9
+```
+The preceding step in this stage is performing hostapd configuration. 
+In order to perform hostapd configuration, type in the following command in the terminal.
+```bash
+sudo gedit/etc/hostapd.conf
+```
+To properly configure **hostapd**, add the the following code to our configuration file:
+```
+# Set interface
+interface=wlan0
+# Set driver to
+driver=nl80211
+# Set your desired ssid(Wi-Fi name)
+ssid=Wi-FiConnection
+# Set the access point hardware mode to 802.11g
+hw_mode=g
+# Select WIFI channel
+ channel=6
+# Ensure to enable only WPA2 
+wpa_passphrase=Wi-Fi connection
+```
+You can change  your`SSID` name and password with your desired choice. The above configuration will create a hotspot called *Wi-Fi connection* with  a password.
+>N/B:  You can create the configuration file anywhere you want  named  **`hotspot.sh`** using an editor of your choice.
+
+After completing the configuration, we shall stop the running services by  typing in the commands below:
+```bash
+sudo service dnsmasq stop
+```
+```bash
+sudo service hostapd stop
+```
+For a wired connection, you will be required to change **ppp0** to **eth0** or any other number.
+
```

Comment:
What's this `ppp0` and `eth0`? You haven't mentioned them anywhere in your tutorial.